docs/contribution-guides/Development-Standards-Guide.md

```diff
@@ -226,3 +242,301 @@ to just leave it as a public field without a property. As always, make sure to d
 6. Most string or numeric literals should be constants.
 7. Don't define constants that have the same value in multiple places. There should only ever be one place you need to change if you ever need to change a constant's value.
         
+# Writing efficent and maintainble code
+
+Unitystation is a gigantic and ambitious project that holds a lot of content and systems that can get tangled together, to avoid things tangling each other; we highly recommend that you write your code in a modular way that's easy for developers to maintain and and expand upon.
+
+Unity has a great talk on how to do just that in this video: [How to adopt a modular code workflow for your team | Unite 2022](https://youtu.be/nk3gHIZZ5Rg)
+
+Below are good practices and explainations on what to do, so you can avoid having your code refactored/reworked 5 months later and make the lives of other contributors much easier.
+
+## 0. In-code documentation
+
+* You can avoid confusing others (and even yourself) by naming your functions and variables in a clear and descriptive way.
+
+```cs
+//bad code
+void doThing(bool b)
+{
+    state = b;
+}
+```
+
+```cs
+//good code
+private void ThisFunctionWillDoA_B_And_C(bool newState)
+{
+    state = newState;
+}
+```
+
+* Explain what your function does step by step whenever possible. Your code might seem obvious to you, but for other people; that may not be the case.
+
+```cs
+//Example from NightVisionGoggle.cs
+
+/// <summary>
+/// Checks if the item is in the correct ItemSlot which is the eyes.
+/// Automatically returns false if null because of the "is" keyword and null propagation.
+/// </summary>
+private bool IsInCorrectNamedSlot()
+{
+	return pickupable.ItemSlot is { NamedSlot: NamedSlot.eyes };
```

Comment:
is stuff might be confusing for new devs

Reply:
I don't think it can be confusing because the whole point of this section is to demonstrate that comments allow everyone to understand what exactly the code is doing.

Reply:
Could work, but also brings up a good point try not to use code that is too confusing in the first place

Reply:
But this code isn't confusing to start with, and the quirks of it are explained in the docs + the whole point of this section is to tell people the usefulness of code docs.

Bod, respectfully. You're the last one who should have a say or talk about any of this, since the whole reason I've added this was because of **you** specifically.